bankroll/converter/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .modelconverter import dataframeForModelObjects
```

Comment:
Just speaking structurally, I feel like it's a bit weird to have a package named `bankroll.converter.modelconverter`, when we could have `bankroll.model.converter`, y'know?

If you want to make sure that the imported functions are namespaced like "converter" functions, you could structure the imports as so:

```py
import bankroll.model.converter as converter

converter.dataframeForModelObjects(...)
```